ereol_inspiration/ereol_inspiration.features.field_instance.inc

```diff
@@ -117,15 +231,17 @@ function ereol_inspiration_field_default_field_instances() {
       'module' => 'paragraphs',
       'settings' => array(),
       'type' => 'paragraphs_embed',
-      'weight' => 5,
+      'weight' => 7,
     ),
   );
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('App feed color');
+  t('App feed image');
   t('Category query');
   t('Elementer');
-  t('Query for the main category');
+  t('Søgestreng for hovedkategorien');
```

Comment:
All strings should be in english in the code

Reply:
Fixed.